translate/public/locale/en-US/translate.ftl

```diff
@@ -369,13 +369,13 @@ history-Translation--span-copied =
     .title = Copied ({ $machinerySources })
 
 history-translation--approved =
-    .title = Approved by { $user }
+    .title = Approved by { $user } on { $reviewedDate }
 history-translation--approved-anonymous =
-    .title = Approved
+    .title = Approved on { $reviewedDate }
 history-translation--rejected =
-    .title = Rejected by { $user }
+    .title = Rejected by { $user } on { $reviewedDate }
 history-translation--rejected-anonymous =
-    .title = Rejected
+    .title = Rejected on { $reviewedDate }
```

Comment:
Did you consider using the Fluent `DATETIME()` function for these? With that, you wouldn't need the `customDateString()` function, and have here instead something like this:

```suggestion
    .title = Rejected on { DATETIME($reviewedDate, dateStyle:"long", timeStyle:"medium") }
```

Reply:
Much more succinct and elegant! Thank you for pointing that out